plugins/package-managers/python/src/main/kotlin/Uv.kt

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (C) 2022 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.plugins.packagemanagers.python
+
+import java.io.File
+
+import org.apache.logging.log4j.kotlin.logger
+
+import org.ossreviewtoolkit.analyzer.PackageManager
+import org.ossreviewtoolkit.analyzer.PackageManagerFactory
+import org.ossreviewtoolkit.downloader.VersionControlSystem
+import org.ossreviewtoolkit.model.Identifier
+import org.ossreviewtoolkit.model.Project
+import org.ossreviewtoolkit.model.ProjectAnalyzerResult
+import org.ossreviewtoolkit.model.Scope
+import org.ossreviewtoolkit.model.config.AnalyzerConfiguration
+import org.ossreviewtoolkit.model.config.Excludes
+import org.ossreviewtoolkit.plugins.api.OrtPlugin
+import org.ossreviewtoolkit.plugins.api.PluginDescriptor
+import org.ossreviewtoolkit.plugins.packagemanagers.python.utils.PythonInspector
+import org.ossreviewtoolkit.plugins.packagemanagers.python.utils.getPythonVersion
+import org.ossreviewtoolkit.plugins.packagemanagers.python.utils.getPythonVersionConstraint
+import org.ossreviewtoolkit.plugins.packagemanagers.python.utils.toOrtPackages
+import org.ossreviewtoolkit.plugins.packagemanagers.python.utils.toPackageReferences
+import org.ossreviewtoolkit.utils.common.CommandLineTool
+import org.ossreviewtoolkit.utils.common.safeDeleteRecursively
+import org.ossreviewtoolkit.utils.ort.createOrtTempFile
+
+internal object UvCommand : CommandLineTool {
+    override fun command(workingDir: File?) = "uv"
+
+    override fun transformVersion(output: String) = output.substringAfter("version ").removeSuffix(")")
+}
+
+/**
+ * [Uv](https://github.com/astral-sh/uv) package manager for Python.
+ */
+@OrtPlugin(
+    displayName = "Uv",
+    description = "An extremely fast Python package and project manager.",
+    factory = PackageManagerFactory::class
+)
+class Uv(override val descriptor: PluginDescriptor = UvFactory.descriptor, private val config: PipConfig) :
+    PackageManager("Uv") {
+    companion object {
+        /**
+         * The name of the build system requirements and information file used by modern Python packages.
+         */
+        internal const val PYPROJECT_FILENAME = "pyproject.toml"
+    }
+
+    override val globsForDefinitionFiles = listOf("uv.lock")
```

Comment:
This line triggered https://github.com/oss-review-toolkit/ort/pull/10051/commits/cc7059a0ecc3521c47f74ef48b4e9e67b2382976 on my side. Once that PR is merged, please take over the final wording of that comment here.

Reply:
Done